setup.py

```diff
@@ -27,7 +27,6 @@
     "numpy>=1.16.3",
     "scipy>=1.0.0",
     "networkx>=2.0",
-    "tensorflow>=1.3.0,<1.7",
     "quantum-blackbird"
```

Comment:
No minimum version number for quantum-blackbird?

Reply:
is it because the api is not set yet?

Reply:
Yep, essentially. I can preliminarily make it `>=0.2.0`?